_integration-test/run.sh

```diff
@@ -41,7 +41,7 @@ echo "${_endgroup}"
 echo "${_group}Starting Sentry for tests ..."
 # Disable beacon for e2e tests
 echo 'SENTRY_BEACON=False' >> $SENTRY_CONFIG_PY
-echo y | $dcr web createuser --force-update --superuser --email $TEST_USER --password $TEST_PASS
+$dcr web createuser --no-input --force-update --superuser --email $TEST_USER --password $TEST_PASS
```

Comment:
I thought this was set through the `CI` variable 🤔

Reply:
Thinking of this?

https://github.com/getsentry/self-hosted/blob/518a9ec1c0f0272650765a8265db52baa38a4382/install/set-up-and-migrate-database.sh#L3-L4